test/hisi_sec_test/test_hisi_sec.c

```diff
@@ -1744,8 +1745,12 @@ int main(int argc, char *argv[])
 	if (!strcmp(argv[3], "-optype")) {
 		g_direction = strtoul((char*)argv[4], NULL, 10);
 		if (algtype_class == DIGEST_CLASS) {
```

Comment:
Why does the mode number in the range of 0, 1 and 3? Do we need mode number 2?

Reply:
we should use getopt_long to do this

Reply:
maybe 2 will be used for other mode ：）